### PR TITLE
Implemented Tabs Component

### PR DESCRIPTION
**Pull request by IlanSirisky**
Generic tab, can receive function, text, image and default value isActive.

### CODE REVIEW

File: src/components/Tab/Tab.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  onClick?: (active: boolean) => void;
+  style?: CSSProperties;
+}
+
+const Tab = ({ text = "list", active = false, onClick, img, style }: TabProps) => {
+  const [isActive, setIsActive] = useState(active);
+
+  const handleClick = () => {
+    setIsActive(!isActive);
+    if (onClick) onClick(!isActive);
```
Comment (ZivCohenMoveo) [inline src/components/Tab/Tab.tsx]: you can change it to:
```suggestion
     onClick?.(!isActive);
```

File: src/components/Tab/Tab.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  active?: boolean;
+  onClick?: (active: boolean) => void;
+  style?: CSSProperties;
+}
+
+const Tab = ({ text = "list", active = false, onClick, img, style }: TabProps) => {
+  const [isActive, setIsActive] = useState(active);
+
+  const handleClick = () => {
+    setIsActive(!isActive);
```
Comment (ZivCohenMoveo) [inline src/components/Tab/Tab.tsx]: just to be sure that you're aware to the fact that setState is async so in the next line the `isActive` won't be the new value

**Review by ZivCohenMoveo — CHANGES_REQUESTED** [review]

File: src/components/Tab/Tab.tsx (hunk 2 above)
Comment (IlanSirisky) [reply src/components/Tab/Tab.tsx]: Updated to use prev value

File: src/components/Tab/Tab.tsx (hunk 1 above)
Comment (IlanSirisky) [reply src/components/Tab/Tab.tsx]: Made change.

**Pull request merged**